docs/setup/install/deb-init.asciidoc

```diff
@@ -0,0 +1,21 @@
+==== Running Kibana with SysV `init`
+
+Use the `update-rc.d` command to configure Kibana to start automatically
+when the system boots up:
+
+[source,sh]
+--------------------------------------------------
+sudo update-rc.d kibana defaults 95 10
+--------------------------------------------------
+
+Kibana can be started and stopped using the `service` command:
```

Comment:
How about: You can start and stop {kib} using the `service` command.  If that works you can make this change throughout the doc.